source/containers/Form/hooks/textReplacement.ts

```diff
@@ -105,15 +106,15 @@ const replaceDates = (descriptor: string[], period?: FormPeriod): string => {
 
   if (descriptor[1] === "currentYear") {
     const year = new Date(
```

Comment:
If currentDate is a date object you could (possibly) just do:

```js
const year = currentDate.getFullYear(); 
```

Reply:
Fixed.